src/@next/Banner/BannerStyle.ts

```diff
@@ -0,0 +1,93 @@
+import styled from 'styled-components';
+import * as Breakpoints from '../utilities/breakpoints';
+import { borderRadius8 } from '../utilities/borderRadius';
+import { Blue, Green, Neutral, Orange, Red } from '../utilities/colors';
+
+export const StyledCloseIconWrapper = styled.div`
+  cursor: pointer;
+  svg {
+    right: 0;
+    top: 0 !important;
```

Comment:
would it be worth writing it this way instead? https://styled-components.com/docs/faqs#how-can-i-override-styles-with-higher-specificity

Reply:
Oh, didn't know about this, let me try. Thanks